tests/Application/config/sylius/1.8/packages/_sylius.yaml

```diff
@@ -0,0 +1,39 @@
+imports:
```

Comment:
This file is inconsistent with PluginSkeleton? 🤔 
https://github.com/Sylius/PluginSkeleton/blob/1.10/tests/Application/config/sylius/1.8/packages/_sylius.yaml